domene/src/main/java/no/nav/foreldrepenger/mottak/task/SikkerhetsnettOnceTask.java

```diff
@@ -0,0 +1,55 @@
+package no.nav.foreldrepenger.mottak.task;
+
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import jakarta.enterprise.context.Dependent;
+import jakarta.inject.Inject;
+import no.nav.foreldrepenger.journalføring.domene.JournalpostId;
+import no.nav.foreldrepenger.journalføring.oppgave.Journalføringsoppgave;
+import no.nav.foreldrepenger.mottak.task.sikkerhetsnett.SikkerhetsnettJournalpost;
+import no.nav.foreldrepenger.mottak.task.sikkerhetsnett.SikkerhetsnettKlient;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTask;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskData;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskGruppe;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskHandler;
+import no.nav.vedtak.felles.prosesstask.api.ProsessTaskTjeneste;
+
+@Dependent
+@ProsessTask(value = "vedlikehold.once.sikkerhetsnett", maxFailedRuns = 1)
+public class SikkerhetsnettOnceTask implements ProsessTaskHandler {
+
+    private static final Logger LOG = LoggerFactory.getLogger(SikkerhetsnettOnceTask.class);
+
+    private final Journalføringsoppgave journalføringsoppgave;
+    private final SikkerhetsnettKlient sikkerhetsnettKlient;
+    private final ProsessTaskTjeneste prosessTaskTjeneste;
+
+    @Inject
+    public SikkerhetsnettOnceTask(Journalføringsoppgave journalføringsoppgave,
+                                  SikkerhetsnettKlient sikkerhetsnettKlient,
+                                  ProsessTaskTjeneste prosessTaskTjeneste) {
+        this.journalføringsoppgave = journalføringsoppgave;
+        this.sikkerhetsnettKlient = sikkerhetsnettKlient;
+        this.prosessTaskTjeneste = prosessTaskTjeneste;
+    }
+
+    @Override
+    public void doTask(ProsessTaskData prosessTaskData) {
+        var åpneJournalposterUtenOppgave = sikkerhetsnettKlient.hentÅpneJournalposterEldreEnn(0).stream()
```

Comment:
Er det korrekt at det skal være 0 dager? Jeg trodde opprinnelig at vi skulle gjøre dette for alt som er eldre enn 2 dager.

Reply:
Har slettet akkurat den - den var laget i tilfelle de ikke fikk joarkhendelser online